src/main/java/com/example/ordermanagementsystemapi/model/Order.java

```diff
@@ -0,0 +1,32 @@
+package com.example.ordermanagementsystemapi.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+
+import java.util.Collection;
+
+
+@Data
+@NoArgsConstructor
+@Entity
+@NonNull
+public class Order {
+    @Id
```

Comment:
How is Order connected to Product in DB? I mean need to use many-to-many like     @ManyToMany